packages/waku/src/lib/plugins/vite-plugin-deploy-aws-lambda.ts

```diff
@@ -15,30 +15,42 @@ const getServeJsContent = (
   distPublic: string,
   srcEntriesFile: string,
 ) => `
-import path from 'node:path';
-import { existsSync, readFileSync } from 'node:fs';
-import { serverEngine, importHono, importHonoNodeServerServeStatic, importHonoAwsLambda } from 'waku/unstable_hono';
+import path from "node:path";
```

Comment:
We change prettier config, only for this.
https://github.com/dai-shi/waku/blob/5e183b8ebf693379ce9cd675a971b02ebd562c50/package.json#L58

Reply:
OK. Sorry about that. I though vscode would use the project settings when I pasted into an unsaved editor buffer and ran the formatter, but I guess not. I copied to the file system and used `pnpx prettier -c --write` instead. Better now. https://github.com/dai-shi/waku/pull/923/commits/fc337de6fa1b2a997d7c5c8d32bd4d8f6690dc3c